app/models/user.rb

```diff
@@ -160,4 +162,10 @@ def can_comment?(subject)
     return true if github_app_authorized? && subject.repository.commentable?
     return false
   end
+
+  def create_default_pinned_searches 
+    pinned_searches.create(query: 'state:closed,merged archived:false', name: 'Archivable')
+    pinned_searches.create(query: 'type:pull_request state:open status:success', name: 'Mergeable')
```

Comment:
Should this also include archived:false?

Reply:
Fixed, good spot!

Reply:
++ thanks @pradyunsg